GrassChoppers/Modules/Home/HomeLogicController.swift

```diff
@@ -1,11 +1,12 @@
 protocol HomeLogicControlling: class {
     var delegate: HomeLogicControllingDelegate? { get set }
     
+    func viewDidLoad()
```

Comment:
This protocol defines what it means to be a `HomeLogicController`. Same concept as for the [data manager](https://github.com/jwitcig/GrassChoppers/pull/2/files#r463916128), but for the logic controller.